cmd/main.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/openshift/assisted-service/internal/cluster/validations"
 	"github.com/openshift/assisted-service/internal/common"
 	"github.com/openshift/assisted-service/internal/connectivity"
-	adiiov1alpha1 "github.com/openshift/assisted-service/internal/controller/api/v1alpha1"
+	agent_install_v1beta1 "github.com/openshift/assisted-service/internal/controller/api/v1beta1"
```

Comment:
I was following @flaper87's footsteps here, with:
https://github.com/openshift/assisted-service/blob/834faecd31fbccaa9716ca020da405a2d13296f8/internal/controller/controllers/bmh_agent_controller.go#L25

I think that makes it more readable.

Reply:
Since this is an alias for a package name, and the go spec even [refers to this token as "PackageName"](https://golang.org/ref/spec#Import_declarations), it's common for aliases to follow the [naming conventions for packages](https://blog.golang.org/package-names). So a snake-case name can be a surprising way to access a package.

Reply:
ack. done.